ml-algorithms/src/main/java/org/opensearch/ml/engine/indices/MLIndicesHandler.java

```diff
@@ -41,6 +41,9 @@ public class MLIndicesHandler {
     ClusterService clusterService;
     Client client;
     private static final Map<String, AtomicBoolean> indexMappingUpdated = new HashMap<>();
+    // This setting is for index update only, so only dynamic settings should be contained!
+    // For index creation setting, use INDEX_SETTINGS directly.
+    public static final Map<String, Object> UPDATED_INDEX_SETTINGS = Map.of("index.auto_expand_replicas", "0-1");
```

Comment:
Should we keep `UPDATED_INDEX_SETTINGS` and `INDEX_SETTINGS` both at the same place? This class name seems suitable for this.

Reply:
In fact I totally agree with this idea, but the issue here is for the INDEX_SETTING variable we have to keep it in the memory interface, or it will cause a circular dependency. So there are two ways, the first one is to store both of them in the memory interface, and the second one is to keep as it is. Keeping in the memory seems more tidier, but because MLIndicesHandler is the only place to use this update setting, to keep it as near the line we use it as possible seems also not a bad idea. Which one do you prefer? @dhrubo-os

Reply:
There's a utils folder in common module. What do you think about creating a `indexUtils` class and add these two variables there? All other modules use common module. What do you think?

Reply:
Agree, refactored.